presto-password-authenticators/src/main/java/io/prestosql/plugin/password/ldap/LdapAuthenticator.java

```diff
@@ -248,7 +240,7 @@ private String validateGroupMembership(String user, DirContext context)
     private NamingEnumeration<SearchResult> searchGroupMembership(String user, DirContext context)
             throws NamingException
     {
-        String userBase = userBaseDistinguishedName.orElseThrow(VerifyException::new);
+        String userBase = userBaseDistinguishedName.get();
```

Comment:
I agree, but we have several other raw `get()` calls here, so it seems better to be consistent. Hiding it in this way is kind of a code smell -- it would be better to see if the code can be structured to avoid this. Maybe by creating some helper classes to hold the state.

In Java 10+, `get()` should be replaced with `orElseThrow()` to be explicit.